src/renderList.js

```diff
@@ -0,0 +1,14 @@
+async function renderList() {
```

Comment:
- [ ] Please, use arrow function according to ES6 structure in place function word, this will help to make your code with better practices. 🐶